Java_codes/HollowInverted.java

```diff
@@ -0,0 +1,40 @@
+import java.util.Scanner;
+public class HItangle
```

Comment:
Can you please find some other directory for your code since Java_codes is not really a relevant directory name for a project with only java codes